consent/strategy_default.go

```diff
@@ -745,10 +745,38 @@ func (s *DefaultStrategy) issueLogoutVerifier(w http.ResponseWriter, r *http.Req
 	// logout flow.
 	redir := s.c.LogoutRedirectURL().String()
 
-	// The hint must be set if it's an RP-initiated logout flow.
-	hint := r.URL.Query().Get("id_token_hint")
-	state := r.URL.Query().Get("state")
-	requestedRedir := r.URL.Query().Get("post_logout_redirect_uri")
+	var hint string
+	var state string
+	var requestedRedir string
+
+	err := r.ParseForm()
+	if err != nil {
+		return nil, errors.WithStack(fosite.ErrInvalidRequest.
+			WithHint(
+				fmt.Sprintf(
+					`Logout failed because the "%s" request could not be parsed`,
+					r.Method,
+				),
+			),
+		)
+	}
+
+	hint = r.Form.Get("id_token_hint")
+	state = r.Form.Get("state")
+    fmt.Printf("Received state: %s", state)
+	requestedRedir = r.Form.Get("post_logout_redirect_uri")
+
+	if len(hint) != 0 && (r.Method != http.MethodGet && r.Method != http.MethodPost) {
```

Comment:
This is not necessary, just register `GET` and `POST` here (needs to be done anyways for this to work :) ): https://github.com/ory/hydra/blob/2dbe523b796ec77796f1cfce1042223011252b16/oauth2/handler.go#L83